src/glmfit.jl

```diff
@@ -469,59 +453,6 @@ function StatsBase.fit!(m::AbstractGLM;
     _fit!(m, maxiter, minstepfac, atol, rtol, start)
 end
 
-function StatsBase.fit!(m::AbstractGLM,
-                        y;
-                        wts=nothing,
-                        offset=nothing,
-                        maxiter::Integer=30,
-                        minstepfac::Real=0.001,
-                        atol::Real=1e-6,
-                        rtol::Real=1e-6,
-                        start=nothing,
-                        kwargs...)
-    if haskey(kwargs, :maxIter)
-        Base.depwarn("'maxIter' argument is deprecated, use 'maxiter' instead", :fit!)
-        maxiter = kwargs[:maxIter]
-    end
-    if haskey(kwargs, :minStepFac)
-        Base.depwarn("'minStepFac' argument is deprecated, use 'minstepfac' instead", :fit!)
-        minstepfac = kwargs[:minStepFac]
-    end
-    if haskey(kwargs, :convTol)
-        Base.depwarn("'convTol' argument is deprecated, use `atol` and `rtol` instead", :fit!)
-        rtol = kwargs[:convTol]
-    end
-    if haskey(kwargs, :verbose)
-        Base.depwarn("""`verbose` argument is deprecated, use `ENV["JULIA_DEBUG"]=GLM` instead.""", :fit!)
-    end
-    if !issubset(keys(kwargs), (:maxIter, :minStepFac, :convTol, :verbose))
-        throw(ArgumentError("unsupported keyword argument"))
-    end
-    if haskey(kwargs, :tol)
-        Base.depwarn("`tol` argument is deprecated, use `atol` and `rtol` instead", :fit!)
-        rtol = kwargs[:tol]
-    end
-
-    r = m.rr
-    V = typeof(r.y)
-    r.y = copy!(r.y, y)
-    isa(wts, Nothing) || copy!(r.wts, wts)
-    isa(offset, Nothing) || copy!(r.offset, offset)
-    initialeta!(r.eta, r.d, r.l, r.y, r.wts, r.offset)
-    updateμ!(r, r.eta)
-    fill!(m.pp.beta0, 0)
-    m.fit = false
-    m.maxiter = maxiter
-    m.minstepfac = minstepfac
-    m.atol = atol
-    m.rtol = rtol
-    if dofit
```

Comment:
This variable isn't defined so clearly nobody has hit this method in a very long time